FeatureAreaFilterForInnerLoop.props

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
We should probably come at this from the other direction. By default InnerLoop should build a very bare-bones MUX with almost everything excluded. An IC can then add the specific feature they wish to work on to the build.

Reply:
That can trip up someone who does not know about this feature no ?

Reply:
It seems like the most common case is that I am working on a particular feature (e.g. ColorPicker), so I would only want to build that feature. It would be easier to enable ColorPicker than disable everything except ColorPicker. That seems like the most common use case for this, so we should make that easy.

Reply:
I added a list of entries, we can just enable the ones that you care about.